### ISSUE TITLE
Python3 compatability

### ISSUE THREAD

**Issue by robinandeer** [robinandeer/cosmid#19]

**Comment by robinandeer** [comment]
2to3 seems to work right off the bat (not introducing weird changes at least)
Read up more on that project

**Comment by mdshw5** [comment]
Looking at your code you should be able to use some `from six.moves import ...` strategically after your 2to3 conversion to provide back-compatibility for py2.6 and 2.7.

**Comment by robinandeer** [comment]
Great! That package was new to me. I knew about `from __future__ ...` but would rather like to develop in Python 3.x directly which I suppose "six" enables.

**Comment by mdshw5** [comment]
Yes, that's the idea. It's good to have some unit tests that you can run and then use something like [tox](http://tox.readthedocs.org/en/latest/) to run your tests under versions of py2 and py3.
